mlir/include/mlir/IR/OperationSupport.h

```diff
@@ -1322,7 +1322,14 @@ struct OperationEquivalence {
     // When provided, the location attached to the operation are ignored.
     IgnoreLocations = 1,
 
-    LLVM_MARK_AS_BITMASK_ENUM(/* LargestValue = */ IgnoreLocations)
+    // When provided, the dictionary attributes attached to the operation are
+    // ignored.
+    IgnoreDictionaryAttrs = 2,
```

Comment:
I think this should be named IgnoreDiscardableAttrs? And you'll want to clarify that inherent attributes are part of "properties"

Or you need three flags: inherent attributes, discardable attributes, and properties?